tools/diamond.py

```diff
@@ -27,7 +28,9 @@ class Diamond(tools.Tool):
 
     def __init__(self, install_methods=None):
         if not install_methods:
-            install_methods = [DownloadAndBuildDiamond(URL, os.path.join(DIAMOND_DIR, 'bin', 'diamond'))]
+            install_methods = [
+                DownloadAndBuildDiamond(URL, os.path.join(DIAMOND_DIR, 'bin', 'diamond')),
+                tools.CondaPackage("diamond", version=CONDA_VERSION)]
```

Comment:
You are right. This is not ready yet because the `CondaPackageVersion` spec is in another PR.